TrafficCapture/captureProtobufs/src/main/proto/TrafficCaptureStream.proto

```diff
@@ -69,6 +69,8 @@ message TrafficObservation {
 message TrafficStream {
   string connectionId = 1; // persistent identifier for all stream objects within a single connection
   string nodeId = 5; // unique namespace such as a nodeId or computer mac address
+  optional int32 requestCount = 6;
+  optional bool lastObservationWasRequestRead = 7;
```

Comment:
Do we have any concern with responses that take multiple TrafficStreams? I read this as though we are only concerned with requests that span multiple TrafficStreams

Reply:
I think they'll always be optional.  In subsequent commits they're optional.  With protobuf, an optional value that isn't present in the binary will be interpreted as the default value for the type, meaning 0 and false.  That's what I want anyway and lets us remain backward compatible with existing streams.

Reply:
I see and yes this was more of a replayer concern I had. That said getting the full requests would be priority and at least initially we could do a best effort approach to get the full response but there are at least restart cases where we plan to do improvement in the future

Reply:
Notice that there are 3 different ways that we'll terminate a response.  
1. The next observation received is a close
2. The next observation received is the beginning of a new request (read observations)
3. There is no next TrafficStream and the liveStreams object (an ExpiringTrafficStreamMap field in CapturedTrafficToHttpTransactionAccumulator) expires the TrafficStream.

If the proxy is expiring connections after it's configured period.  Either there's a misconfiguration that the window is too short, or we're missing data coming in from Kafka.  Both of which should involve investigation of the environment.